src/components/Home.js

```diff
@@ -0,0 +1,50 @@
+import { PropTypes } from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+
+import { BsArrowRightCircle } from 'react-icons/bs';
+import { motion } from 'framer-motion';
+
```

Comment:
- [ ] It would be better if our app design follows the 2 column layout for mobile devices. However, on mobile devices we have a 1 column layout :man_technologist: Let us kindly address this